docs/components/menu/index.html

```diff
@@ -0,0 +1,156 @@
+      <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+    </hx-menu>
+  </div>
+  {% code 'html' %}
+    <hx-disclosure class="hxBtn" aria-controls="menuId">
+      <hx-icon type="cog"></hx-icon>
+      Actions
+      <hx-icon class="menu-icon" type="angle-down"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="menuId">
+      <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+    </hx-menu>
+  {% endcode %}
+</section>
+<!--
+<section>
+<h2 class="hxSectionTitle" id="cog-menu">Cog Menu</h2>
+  <div class="demo">
+    <hx-disclosure class="hxBtn btn-cog" aria-controls="cogmenuId">
+      <hx-icon class="icon" type="cog"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="cogmenuId" >
+      <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+    </hx-menu>
+  </div>
+  {% code 'html' %}
+    <hx-disclosure class="hxBtn btn-cog" aria-controls="cogmenuId">
+      <hx-icon class="icon" type="cog"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="cogmenuId" >
+      <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+    </hx-menu>
+  {% endcode %}
+</section>
+-->
+
+<section>
+  <h2 class="hxSectionTitle" id="advanced-menu">Advanced Menu</h2>
+  <div class="demo">
+    <hx-disclosure class="hxBtn" aria-controls="advancedmenuId">
+      <hx-icon type="cog"></hx-icon>
+      Actions
+      <hx-icon class="menu-icon" type="angle-down"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="advancedmenuId">
+      <hx-menuitem-group>
+        <header>TITLE</header>
+        <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+        <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+        <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+      </hx-menuitem-group>
+      <hx-menuitem-group>
+        <header>TITLE</header>
+        <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+        <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+        <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+      </hx-menuitem-group>
+    </hx-menu>
+  </div>
+  {% code 'html' %}
+    <hx-disclosure class="hxBtn" aria-controls="advancedmenuId">
+      <hx-icon type="cog"></hx-icon>
+      Actions
+      <hx-icon class="menu-icon" type="angle-down"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="advancedmenuId" position="bottom">
```

Comment:
this `position="bottom"` isn't needed

Reply:
EDIT: I see that maybe we do allow for `position` to be set now?

If so, we should at least have this example markup match the component above. They should either both have a non-default position set, or not.

Reply:
good catch, removed